x-pack/test/observability_functional/apps/observability/alerts/rules_page.ts

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default ({ getService }: FtrProviderContext) => {
+  const esArchiver = getService('esArchiver');
+  const testSubjects = getService('testSubjects');
+  const supertest = getService('supertest');
+  const find = getService('find');
+
+  const retry = getService('retry');
+
+  async function createRule(rule: any): Promise<string> {
+    const ruleResponse = await supertest
+      .post('/api/alerting/rule')
+      .set('kbn-xsrf', 'foo')
+      .send(rule);
+    expect(ruleResponse.status).to.eql(200);
+    return ruleResponse.body.id;
+  }
+  async function deleteRuleById(ruleId: string) {
+    const ruleResponse = await supertest
+      .delete(`/api/alerting/rule/${ruleId}`)
+      .set('kbn-xsrf', 'foo');
+    expect(ruleResponse.status).to.eql(204);
+    return true;
+  }
+
+  const getRulesList = async (tableRows: any[]) => {
+    const rows = [];
+    for (const euiTableRow of tableRows) {
+      const $ = await euiTableRow.parseDomContent();
+      rows.push({
+        name: $.findTestSubjects('rulesTableCell-name').find('a').text(),
+        enabled: $.findTestSubjects('rulesTableCell-ContextStatus').find('button').attr('title'),
+      });
+    }
+    return rows;
+  };
+
+  describe('Observability Rules page', function () {
+    this.tags('includeFirefox');
+
+    const observability = getService('observability');
+
+    before(async () => {
+      await esArchiver.load('x-pack/test/functional/es_archives/observability/alerts');
+      await esArchiver.load('x-pack/test/functional/es_archives/infra/metrics_and_logs');
+      await observability.alerts.common.navigateWithoutFilter();
+    });
+
+    after(async () => {
+      await esArchiver.unload('x-pack/test/functional/es_archives/observability/alerts');
+      await esArchiver.unload('x-pack/test/functional/es_archives/infra/metrics_and_logs');
+    });
+
+    describe('Feature flag', () => {
+      // Related to the config inside x-pack/test/observability_functional/with_rac_write.config.ts
+      it('Link point to O11y Rules pages by default or when "xpack.observability.unsafe.rules.enabled: true"', async () => {
```

Comment:
@fkanout If it is not much of a hassle it would be nice to have another test where `xpack.observability.unsafe.rules.enabled: false` and link points to stack management page.

Reply:
I thought about that, and I asked the Kibana core team about it. The only way is to have a separate server-test-runner file to do that.